packages/integration/test/utils/config.ts

```diff
@@ -40,12 +41,14 @@ const DEFAULT_LOCAL_CONFIG = {
       confirmations: 1,
       subgraph: "http://localhost:8010/subgraphs/name/connext/nxtp",
       transactionManagerAddress: "0x8CdaF0CD259887258Bc13a92C0a6dA92698644C0",
+      priceOracleAddress: "0x0000000000000000000000000000000000000000",
```

Comment:
Do we have a solution for local testnet testing? Does AddressZero mock things out?

Reply:
I think It is impossible to have a local testnet testing.

Reply:
Hmm can we do it so that routers do not need to update the config? We can use our deployed priceOracle, and if it's not deployed that means there is no gas fee? That way we can simply use the existence of the contract to check for the chain.